roundedimageview/src/main/java/com/makeramen/roundedimageview/RoundedImageView.java

```diff
@@ -337,12 +387,42 @@ private void updateAttrs(Drawable drawable) {
     }
   }
 
+  private void updateBackgroundAttrs(Drawable drawable) {
+    if (drawable == null) { return; }
+
+    if (drawable instanceof RoundedDrawable) {
+      ((RoundedDrawable) drawable)
+              //It's expected for a Background to fit the drawable to
+              //cover all background, so its ScaleType should be FIT_XY
+              .setScaleType(ScaleType.FIT_XY)
```

Comment:
Is this always true? What happens when the image is FIT_START or FIT_END like in the example? Should we try to match the background to the image's intrinsic dimensions?